<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="EmployeePayrollServiceTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2020-12-19T10:49:15" hostname="DESKTOP-J9DM2GE" time="0.283">
  <properties/>
  <testcase name="givenDateRange_WhenRetrieved_ShouldMatchEmployeeCount" classname="EmployeePayrollServiceTest" time="0.283"/>
  <system-out><![CDATA[Connecting to database: jdbc:mysql://localhost:3306/payroll_service?useSSL=false
Connection is Successful!!!!!!!!com.mysql.jdbc.JDBC4Connection@4d346a1f
Connecting to database: jdbc:mysql://localhost:3306/payroll_service?useSSL=false
Connection is Successful!!!!!!!!com.mysql.jdbc.JDBC4Connection@3bbbaa20
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
